atm.js

```diff
@@ -1,21 +1,45 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+var checkingWithdrawButton = $("#withdrawButton");
+var checkingBalance = $("#checkingBalance");
+var checkingNewDeposit = $("#valueinput");
+var checkingValue = $("#checkingNewDeposit.val()");
+var cBalance= checkingBalance.html().substr(1);
```

Comment:
This is repeated below - it has to be inside the click handler so that the variable is stored after the balance is updated